### PR TITLE
prefer phpunit 5.x on hhvm

### PR DESCRIPTION
**Pull request by eventhorizonpl**
Hi,

HHVM is identified as PHP 5.6 compiler. There are some unsolved problems with tests on hhvm, but using a recent phpunit should be a good start point. How about using 5.1?

Best regards,
Michal

| Q | A |
| --- | --- |
| Bug fix? | yes |
| New feature? | yes |
| BC breaks? | no |
| Deprecations? | no |
| Tests pass? | yes |
| License | MIT |

### CODE REVIEW

File: phpunit [hunk 1]
```diff
@@ -19,7 +19,7 @@ error_reporting(-1);
 require __DIR__.'/src/Symfony/Component/Process/ProcessUtils.php';
 
 // PHPUnit 4.8 does not support PHP 7, while 5.0 requires PHP 5.6+
-$PHPUNIT_VERSION = PHP_VERSION_ID >= 70000 ? '5.0' : '4.8';
+$PHPUNIT_VERSION = ((PHP_VERSION_ID >= 70000) or (PHP_VERSION_ID >= 50600)) ? '5.0' : '4.8';
```
Comment (Tobion) [inline phpunit]: `PHP_VERSION_ID >= 70000` part makes no sense anymore with this condition

**Comment by EVODelavega** [comment]
I've changed that same line to `$PHPUNIT_VERSION = PHP_VERSION_ID >= 50600 ? '5.0' : '4.8';` because travis' HHVM tests kept failing. After changing that the build was successful. Remove the `or` and I think this PR is ready to merge

**Comment by eventhorizonpl** [comment]
@EVODelavega @Tobion 

Done. How about using phpunit 5.1 instead of 5.0?

**Comment by nicolas-grekas** [comment]
Please update the PR with 5.1 so that we can know if it works

**Comment by eventhorizonpl** [comment]
@nicolas-grekas 

done

**Comment by nicolas-grekas** [comment]
:+1:

**Comment by nicolas-grekas** [comment]
Thank you @eventhorizonpl.

**Pull request closed without merging**